src/bika/coa/reports/GreenhillMulti.pt

```diff
@@ -159,558 +168,579 @@
+              <thead>
+                <tr>
+                  <th colspan="4">Sample ID</th>
+                  <tal:ar repeat="model page">
+                    <th colspan="1" class="font-weight-normal">
+                      <div class="text-primary text-center"
+                          tal:content="model/Title"/>
+                    </th>
+                  </tal:ar>
+                </tr>
+                <tr class="noborder" style="border:none !important">
+                  <tal:ar repeat="model page">
+                    <th colspan="4" tal:condition="python: repeat['model'].start" class="noborder">
+                          Client Sample ID
+                    </th>
+                    <th colspan="1" class="font-weight-normal text-center noborder">
+                        <div class="text-center" tal:content="python:model.ClientSampleID or '-'"/>
+                    </th>
+                  </tal:ar>
+                </tr>
+                <tr class="noborder">
+                  <tal:ar repeat="model page">
+                    <th class="noborder" colspan="4" tal:condition="python: repeat['model'].start">
+                          <div class="text-left">Sample Type</div>
+                    </th>
+                    <th colspan="1" class="font-weight-normal text-center noborder">
+                        <div class="text-center"
+                            tal:content="python:model.SampleTypeTitle or '-'"/>
+                    </th>
+                  </tal:ar>
+                </tr>
+                <tr class="noborder">
+                  <tal:ar repeat="model page">
+                  <th class="noborder" colspan="4" tal:condition="python: repeat['model'].start">
+                      <div class="text-left">Sample Point</div>
                   </th>
-                </tal:ar>
-              </tr>
-              <tr class="noborder" style="border:none !important">
-                <tal:ar repeat="model page">
-                  <th colspan="4" tal:condition="python: repeat['model'].start" class="noborder">
-                        Client Sample ID
+                  <th class="font-weight-normal text-center noborder">
+                      <div class="text-center"
+                          tal:content="python:model.SamplePointTitle or '-'"/>
+                  </th>
+                  </tal:ar>
+                </tr>
+                <tr class="noborder">
+                  <tal:ar repeat="model page">
+                  <th class="noborder" colspan="4" tal:condition="python: repeat['model'].start">
+                      <div class="text-left">Sample Condition</div>
                   </th>
                   <th colspan="1" class="font-weight-normal text-center noborder">
-                      <div class="text-center" tal:content="python:model.ClientSampleID or '-'"/>
+                      <tal:sample_condition condition='python: model.SampleCondition'>
```

Comment:
Maybe you can use `getSampleConditionTitle` found in `./bika/lims/content/analysisrequest.py` instead of the two tal conditions